### DISCUSSION TITLE
Usage when installed globally

### DISCUSSION THREAD

**Discussion by Brugman** [permafrost-dev/vue-ray#272] [Q&A] [answered]
> Installation
> When using in a Vue 3 project, you may optionally install the plugin globally in your main.js or app.js file.

> Usage
```
<script setup>
import { raySetup } from 'vue-ray'
const ray = raySetup(); // `ray` is a ref, so you must use `ray.value` within the script tags
</script>
```

Does installing _globally_ mean we don't have to import and store raySetup() in every single component just to do `ray('hello');` in them?

**Comment by patinthehat** [comment, marked as answer]
@Brugman The plugin is primarily for customizing the connection options. It does `provide()` a `$ray` value, but afaik you'd need to use `inject()` to access it.
